test/extensions/filters/network/zookeeper_proxy/config_test.cc

```diff
@@ -60,6 +145,139 @@ stat_prefix: test_prefix
   cb(connection);
 }
 
+TEST(ZookeeperFilterConfigTest, ConfigWithDefaultLatencyThreshold) {
+  const std::string yaml = R"EOF(
+stat_prefix: test_prefix
+default_latency_threshold: "0.15s"
+  )EOF";
+
+  ZooKeeperProxyProtoConfig proto_config;
+  TestUtility::loadFromYamlAndValidate(yaml, proto_config);
+
+  testing::NiceMock<Server::Configuration::MockFactoryContext> context;
+  ZooKeeperConfigFactory factory;
```

Comment:
I see a pattern here; factor out the common context here into a test class and use TEST_F?